docs/pages/switch.js

```diff
@@ -65,7 +255,7 @@ export default function DocsPage({ generatedDocGen }: {| generatedDocGen: DocGen
         >
           <MainSection.Card
             defaultCode={`
-        function SwitchExample() {
+        function SwitchExample4() {
```

Comment:
nit: this is unnecessary — the function components in different examples can all have the same name. only the ids need to be unique

Reply:
Done!